aria/modeling/service_template.py

```diff
@@ -549,9 +549,14 @@ def as_raw(self):
             ('requirement_templates', formatting.as_raw_list(self.requirement_templates))))
 
     def instantiate(self, container):
-        context = ConsumptionContext.get_thread_local()
         from . import models
-        name = context.modeling.generate_node_id(self.name)
+        import pydevd; pydevd.settrace('localhost', suspend=False)
+        if self.nodes:
+            highest_name_suffix = max(self.nodes, key=lambda n: n.name).name.rsplit('_', 1)[1]
```

Comment:
im not sure this would work;
note that `mynode_2` will come after `mynode_11` :|